### PR TITLE
[14.0][IMP] account invoice import simple pdf: add missing values & translation

### PR DESCRIPTION
**Pull request by klodr**
Add some missing values required for some clients.

### CODE REVIEW

**Review by BriceCG — APPROVED** [review]

**Review by BriceCG — APPROVED** [review]

**Review by R-ando — APPROVED** [review]

**Review by razafimiandrisoa — APPROVED** [review]

**Comment by klodr** [comment]
@alexis-via Could review and merge if agreed please ?

**Comment by klodr** [comment]
@florian-dacosta Can you check this ? Thanks @alexis-via seems busy

File: account_invoice_import_simple_pdf/i18n/account_invoice_import_simple_pdf.pot [hunk 1]
```diff
@@ -436,6 +436,78 @@ msgstr ""
 msgid "Missing fixed char."
 msgstr ""
 
+#. module: account_invoice_import_simple_pdf
```
Comment (simahawk) [inline account_invoice_import_simple_pdf/i18n/account_invoice_import_simple_pdf.pot:439]: POT files should be update by https://translation.odoo-community.org/

File: account_invoice_import_simple_pdf/models/account_invoice_import_simple_pdf_invoice_number.py [hunk 2]
```diff
@@ -45,15 +45,17 @@ class AccountInvoiceImportSimplePdfInvoiceNumber(models.Model):
...
-            ("fixed", "Fixed"),
-            ("year2", "Year on 2 digits"),
-            ("year4", "Year on 4 digits"),
-            ("month", "Month (2 digits)"),
-            ("digit", "Digit(s)"),
-            ("letter_upper", "Upper Letter"),
-            ("letter_lower", "Lower Letter"),
-            ("char", "Any Character"),
-            ("space", "Space"),
+            ("fixed", _("Fixed")),
```
Comment (simahawk) [inline account_invoice_import_simple_pdf/models/account_invoice_import_simple_pdf_invoice_number.py:48]: this is not needed for selection fields. The orm will flag these terms as translatable.

**Review by simahawk — CHANGES_REQUESTED** [review]
@alexis-via could you have a look at the logic change?

File: account_invoice_import_simple_pdf/models/account_invoice_import_simple_pdf_invoice_number.py (hunk 2 above)
Comment (klodr) [reply account_invoice_import_simple_pdf/models/account_invoice_import_simple_pdf_invoice_number.py:48]: @simahawk not "will flag " => "should theoretically flag"
In fact it's not working so I've added all missing terms.
Let me know if we can do better and force the orm to work

File: account_invoice_import_simple_pdf/i18n/account_invoice_import_simple_pdf.pot (hunk 1 above)
Comment (klodr) [reply account_invoice_import_simple_pdf/i18n/account_invoice_import_simple_pdf.pot:439]: @simahawk  
Agreed if you now, how to call the bot to do the job, he's supposed to do ;-) 

There is no logical change here we've just add a few case for existing invoice numbers, we've encountered in real life.
That's offering a better security in invoice number detection.

**Comment by simahawk** [comment]
Shall we reopen this?

**Comment by klodr** [comment]
`yes please



Le lun. 1 juil. 2024 à 09:33, Simone Orsi ***@***.***> a
écrit :

**Pull request closed without merging**